colcon-build-and-test/action.yaml

```diff
@@ -90,6 +90,4 @@ runs:
         path: |
           ./build
           ./install
-          ./lcov
-          ./coveragepy
```

Comment:
Where will you store these coverage results?

Reply:
Currently, I don't use these coverage results. So I think we can remove it.
In case we want to take a look the lcov html reports which inside the lcov/ directory, we can use artifacts instead. But since `ros-metrics-reporter` can host the html report, no one willing to download html report directory.

Currently, I don't use the coverage results. So, we can delete it.
If someone wants to see the lcov html reports in the `lcov/` directory, we can use artifacts instead. However, `ros-metrics-reporter` can host html reports, so no one will download the html report directory.

Reply:
Okay, I understand.